### PR TITLE
Transfer: Create SuccessfulTxDialog

### PR DESCRIPTION
**Pull request by rating89us**
New SuccessfulTxDialog component:
- Green circle checkmark animation
- Open folder button (view-only wallet): opens the folder where the transaction file was saved

Normal transaction:
txflownew1

After creating transaction file in view-only wallet (offline transaction signing feature):
image

### CODE REVIEW

**Comment by xiphon** [comment]
> Requires #2895

Why does it require #2895? Couldn't we just move the code showing new dialog to the current PR?

**Comment by rating89us** [comment]
My mistake. I think all code is moved now

File: components/SuccessfulTxDialog.qml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                KeyNavigation.tab: doneButton
+                Keys.enabled: openFolderButton.visible
+                Keys.onReturnPressed: openFolderButton.onClicked
+                Keys.onEnterPressed: openFolderButton.onClicked
+                Keys.onEscapePressed: {
+                    root.close()
+                    root.rejected()
+                }
+                onClicked: {
+                    oshelper.openContainingFolder(walletManager.urlToLocalPath(saveTxDialog.fileUrl))
```
Comment (xiphon) [inline components/SuccessfulTxDialog.qml:167]: Could we use `onCloseCallback` for that?

File: main.qml [hunk 2]
```diff
@@ -1499,6 +1501,19 @@ ApplicationWindow {
         }
     }
 
+    // Transaction successfully sent popup
+    SuccessfulTxDialog {
+        // dynamically change onclose handler
+        property var onCloseCallback
```
Comment (xiphon) [inline main.qml:1507]: Looks like we can drop this

**Comment by xiphon** [comment]
What do you think about adjusting the font size to fit transaction id on a single line?

**Comment by rating89us** [comment]
Font size would need to be too small to fit in current dialog width.
And increasing dialog width would make it too large.
So I prefer the way it is now.

**Review by xiphon — APPROVED** [review]

**Review by selsta — APPROVED** [review]

**Pull request merged**